ModulBot/Cards/ActionButton.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using ModulBot.Database.PostgresRepositories;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.ReplyMarkups;
+using Npgsql;
+using System.Threading.Tasks;
+
+namespace ModulBot.Cards
+{
+    public static class ActionButton
+    {
+        public static async Task DoAction(long chatId, int messageId, string[] buttonCallbackData)
+        {
+            int statId = Convert.ToInt32(buttonCallbackData[0]);
+            int actionType = Convert.ToInt32(buttonCallbackData[1]);
+            var bd = new PostgresStatsRepository();
+            var stat = bd.GetById(statId);
+            string text = "";
+
+            switch (actionType)
+            {
+                case ((int)Constants.ActionTypes.TurnOn):
+                    {
+                        stat.IsActive = true;
+                        text = $"Cтатистика №{statId} включена.";
+                        bd.Update(stat);
+                        break;
+                    }
+                case ((int)Constants.ActionTypes.TurnOff):
+                    {
+                        stat.IsActive = false;
+                        text = $"Cтатистика №{statId} отключена.";
+                        bd.Update(stat);
+                        break;
+                    }
+
+                case ((int)Constants.ActionTypes.Subscribe):
+                    {
+                        stat.Subscribers.Add(chatId);
+                        bd.Update(stat);
+                        text = $"Вы подписались на статистикиу";
+                        break;
+                    }
+
+                case ((int)Constants.ActionTypes.Unsubscribe):
+                    {
+                        stat.Subscribers.Remove(chatId);
+                        bd.Update(stat);
+                        text = $"Вы отписались от статистики";
+                        break;
+                    }
+                case ((int)Constants.ActionTypes.UnsibscribeAll):
+                    {
+                        List<Models.Statistic> stats = new List<Models.Statistic>(bd.GetAll());
+                        for (int i = 0; i < stats.Count; i++  )
+                        {
+                            if (stats[i].Subscribers.Contains(chatId))
+                            {
+                                stats[i].Subscribers.Remove(chatId);
+                                bd.Update(stats[i]);
+                            }
+                        }
+                        
+                        text = "Вы были отписаны от всех статистик";
+                        break;
+                    }
+                case ((int)Constants.ActionTypes.GetStat):
+                    {
+                        text = GetStat(statId);
+                        break;
+                    }
+                case ((int)Constants.ActionTypes.Back):
+                    {
+                        bool isTunestat = (Bot.LastCommand == "tunestat") ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unnecessary Boolean literal(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1125)